src/main/kotlin/com/grepp/nbe1_3_team04/vote/api/request/VoteDateCreateRequest.kt

```diff
@@ -0,0 +1,27 @@
+package com.grepp.nbe1_3_team04.vote.api.request
+
+import com.grepp.nbe1_3_team04.vote.service.request.VoteDateCreateServiceRequest
+import jakarta.validation.constraints.Future
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.Size
+import java.time.LocalDateTime
+
+data class VoteDateCreateRequest(
+    @field:NotBlank(message = "제목은 필수입니다.")
+    @field:Size(max = 50, message = "제목은 50자 이하여야 합니다.")
+    val title: String?,
+    @field:Future(message = "투표 종료 시간은 현재 시간보다 미래의 시간으로 지정해야합니다.")
+    val endAt: LocalDateTime?,
+    @field:Size(min = 1, message = "일정 선택은 필수입니다.")
+    val choices: MutableList<DateChoices>?
+) {
+    fun toServiceRequest(): VoteDateCreateServiceRequest {
+        return VoteDateCreateServiceRequest(
+            title!!,
+            endAt!!,
+            choices!!.stream()
+                .map<LocalDateTime>(DateChoices::choice)
+                .toList()
```

Comment:
코틀린 공부하면서 알아낸 내용인데, 코틀린에서는 Stream API를 거치지 않고 바로 map을 사용할 수 있다고 합니다.
```suggestion
            choices!!.map { it.choice }
```
이렇게 작성할 수 있다고 해요! 그냥 공유만 하려고 합니다!

Reply:
맞습니다.. 코틀린에 스트림대신 맵 사용하는건 알고있었는데 딸깍하다보니 ㅎㅎ ㅋㅋ 다시 리마인드하면서 상기하고가겠습니다 감사합니다